src/components/landing/pages/login/login.scss

```diff
@@ -1,3 +1,3 @@
 .social-button-container {
-  min-width: 30%;
+  width: 100%;
 }
```

Comment:
This class is unnecessary, because the containing column will stretch the button automatically  to full size.

Reply:
I removed the surrounding flex div, which is now obsolete due to the full width of the social buttons and makes the `.social-button-container` unnecessary indeed